lib/webview-helpers.js

```diff
@@ -44,6 +59,14 @@ async function webviewsFromProcs (adb, deviceSocket) {
 // not some other running app
 // TODO: this should be called procFromPid and exist in appium-adb
 async function procFromWebview (adb, webview) {
+  if (webview.match(new RegExp(WEBVIEW_BASE + "(\\d+)")) === null) {
```

Comment:
Use string interpolation instead of concatenating strings
```js
`${WEBVIEW_BASE}(\\d+)`
```

This is also constant, so could be a constant like the other regular expressions.